include/aos/crypto/mbedtls/drivers/aos/driver_wrap.h

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (C) 2024 Renesas Electronics Corporation.
+ * Copyright (C) 2024 EPAM Systems, Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef DRIVER_WRAP_H_
+#define DRIVER_WRAP_H_
+
+#if defined(PSA_CRYPTO_DRIVER_AOS)
+
+#include <aos/common/crypto.hpp>
+
+#include "driver.h"
+
+struct KeyDescription {
```

Comment:
All public definitions should have doxygen comments.

Reply:
Done